docs/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Instead of putting the docs build script in `docs/` and compiling it on its own, I could put it in `src/` and let it get compiled with everything else. However, it seemed nicer to have it separate. I could also rename this directory `scripts/`